### PR TITLE
Memcached json

### PR DESCRIPTION
**Pull request by vetinari**
add json() to deserialize values fetched from memcached

### CODE REVIEW

**Comment by vetinari** [comment]
:+1:

File: tests/test_memcached.py [hunk 1]
```diff
@@ -49,3 +49,14 @@ def test_memcached_stats(monkeypatch):
...
         assert 'uptime' not in ret
+
+
+def test_memcached_json(monkeypatch):
+    mc = MemcachedWrapper(CounterMock().wrapper(), 'localhost')
+    get = MagicMock()
+    get.return_value = '{"foo": 1, "bar": 2}'
+    monkeypatch.setattr('pymemcache.client.base.Client.get', get)
+    ret = mc.json('somekey')
+    assert 'foo' in ret
```
Comment (mohabusama) [inline tests/test_memcached.py]: Can't we assert on the dict here?

**Comment by mohabusama** [comment]
👍

**Comment by vetinari** [comment]
:+1:

**Pull request merged**